bioio_base/metadata_labels.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum, unique
+
+
+@unique
+class MetadataLabels(Enum):
+    DIMENSIONS_PRESENT = "Dimensions_Present"
```

Comment:
what are the allowed dimensions?  anything outside of t,c,z,y,x?

Reply:
We had some discussion about the api here, summarized in the issue ticket... leaning toward making bioio-base define the fields that should be allowed in this "manifest" and not letting individual readers add their own custom data fields to it.
In that spirit, maybe this field should only allow T,C,Z,Y,X and maybe M...although the string "M" in here may not be as interpretable as a separate boolean is_mosaic.

Reply:
https://github.com/bioio-devs/bioio/issues/102